pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Programming Language :: Python :: 3",]
+description = "Interact with and program switches over python"
+dynamic = [ "version", "readme", "dependencies", "optional-dependencies",]
+keywords = []
+name = "switchtool"
+requires-python = ">=3.9"
+
+[options]
+zip_safe = false
+include_package_data = true
+
+[[project.authors]]
+name = "SLAC National Accelerator Laboratory"
+
+[project.license]
+file = "LICENSE.md"
+
+[project.scripts]
+
+[tool.setuptools.packages.find]
+where = [ ".",]
+include = [ "psnet*", ]
```

Comment:
When you install this package, is it accessible under `switchtool` or `psnet`?  The fact that these two are named differently is a bit confusing

Reply:
Kaushik and I chatted about this a bit offline, but to recap: the reason this is confusing is that we might install this via:
`pip install switchtool`

and import it via:
`import psnet`  but `import switchtool` wouldn't work.

I'd advocate for keeping these the same, probably involving renaming the `psnet` internal folder to match `switchtool`

Reply:
I figured that you could just re-route/rename the symlink, but the alternate route is changing the name of the package to not collide with this symlink.  

As in renaming both the package name and import name to something different (`switchtool_gui`?)